tests/data/robots.txt

```diff
@@ -1,10 +1,10 @@
 # robotstxt.org/
 
-User-agent: *
```

Comment:
Maybe also revert all changes to tests. If the change to the package are correct, all tests should still pass without any changes.